src/draggable.js

```diff
@@ -358,6 +358,7 @@
       // trigger dragend event
       if (me.dragEvent.started) {
         options.onDragEnd(element, dragEvent.x, dragEvent.y, e);
+        me.dragEvent.started = false;

```

Comment:
this is already set [here](https://github.com/bcherny/draggable/pull/2/files#diff-2a7114e0f66599e9aceb516f53c6fa0dR374), which is called by #reset below.